app/views/contacts/show.html.erb

```diff
@@ -5,100 +5,87 @@
   });
 </script>
 <% end %>
-<div class="container">
-
-
-
-<div class="contactDiv">
-  <div class="">
-
-
-
-
-<div class="profilPic"  style ="background: url(<%= @contact.avatar_url %>); background-position: center; background-repeat: no-repeat;
-  background-size: cover;">
-
-</div>
-  </div>
-<h1 class="profileName"><%= @contact.full_name %></h1>
-</div>
-
 
+<div class="container">
   <div class="row">
-  <div class="col-7">
-    <div class="container-fluid">
-      <div class="row">
-      <div class="col-4">
-
-        <div class="row font24">
-          Phone:
-        </div>
-        <hr class = "width150"/>
-        <div class="row font24">
-          Email:
-        </div>
-        <hr class = "width150"/>
-        <div class="row font24">
-          Birthday:
-        </div>
-        <hr class = "width150"/>
-        <div class="row font24">
-          Ecclesia:
+    <div class="col-7">
+
+      <div class="row mt-2 mb-3">
```

Comment:
Bootstrap has some handy utility classes. `m` is for `margin`, `t` is for `top` and then you can specify a number `0` through `5` to indicate how much margin. The same set of utility classes exists for padding. Examples:

`mb-5` -> `margin-bottom: 5rem;`
`pl-2` -> `padding-left: 2rem;`
`ml-0` -> `margin-left: 0;`
etc